data/monster/amphibics/emeraldtortoise.lua

```diff
@@ -0,0 +1,109 @@
+local mType = Game.createMonsterType("Emerald Tortoise")
+local monster = {}
+
+monster.name = "Emerald Tortoise"
+monster.description = "an emerald tortoise"
+monster.experience = 12129
+monster.outfit = {
+	lookType = 1550,
+	lookHead = 0,
+	lookBody = 0,
+	lookLegs = 0,
+	lookFeet = 0,
+	lookAddons = 0,
+	lookMount = 0
+}
+
+monster.raceId = 2268
+monster.health = 21040
+monster.maxHealth = 21040
+monster.runHealth = 0
+monster.race = "blood"
+monster.corpse = 39291
+monster.speed = 179
+monster.summonCost = 0
+
+monster.changeTarget = {
+	interval = 2000,
+	chance = 0
+}
+
+monster.flags = {
```

Comment:
The pattern to all monsters is this order:  
 summonable =
	attackable = 
	hostile = 
	convinceable = 
	pushable =
	rewardBoss =
	illusionable = 
	canPushItems =
	canPushCreatures =
	staticAttackChance =
	targetDistance =
	runHealth = 
	healthHidden = 
	isBlockable = 
	canWalkOnEnergy = 
	canWalkOnFire = 
	canWalkOnPoison =

Reply:
Thanks for the reply!

I will fix it.